_lessons/python-scatter/01-profiling.md

```diff
@@ -211,6 +212,17 @@ tell *memory_profiler* which functions you wish to profile.
 We will not cover memory profiler in detail here but more information can be found
 at the page linked above.
 
+## ARM MAP
+
+On NeSI systems another profiler is provide, the [ARM MAP](https://www.arm.com/products/development-tools/server-and-hpc/forge/map) (previous Allinea MAP) profiler, which is part of the `forge` module (as well as the parallel debugger DDT).
+
+In contrast to cProfile, which provides function based profiling information of Python programs, MAP is a commercial product, which can profile  parallel, multi-threaded and single-threaded C/C++, Fortran and F90, as well as Pyhton codes. MAP can be launched with a GUI, which allows the user to navigate thru the code and enables to focus on specific source lines, and also without a GUI for faster execution (see [documentation](https://developer.arm.com/docs/101136/latest/map)).
+
+The GUI can be started after loading `module load forge` and launching `map`. In the profile menu we need to specify the executable (in this case "python"), the arguments (here "scatter.py") and a working directory and if necessary the parallelisation parameters. After running the executable, MAP shows the summary with different metrics.
+[![example-map-scatter](images/MAP_scatter.png)](images/MAP_scatter.png)
```

Comment:
@mand35 the case of the MAP_scatter image is not correct in the link - you either need to rename the file from MAP_scatter.PNG -> MAP_scatter.png, or edit the link to make it MAP_scatter.PNG